internal/command/flags.go

```diff
@@ -19,6 +19,10 @@ var Flags = []cli.Flag{
 		Value: "secrets.yml",
 		Usage: "Path to secrets.yml",
 	},
+	cli.BoolFlag{
+		Name:  "u, up",
+		Usage: "Go up in the directory hierarchy until the secrets file is found",
+	},
```

Comment:
I almost wonder if this flag is dangerous enough to only be allowed with the long-form. Thoughts?

Reply:
I share your concerns actually. I am fine with allowing only the long-form. Should I proceed ?

Reply:
Yes, just use the long form please :)

Reply:
done